db/schema.rb

```diff
@@ -10,10 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2023_09_08_172657) do
+ActiveRecord::Schema[7.1].define(version: 2024_02_19_225156) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "customer_items", id: false, force: :cascade do |t|
+    t.bigint "customer_id", null: false
+    t.bigint "item_id", null: false
+  end
+
+  create_table "customers", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "name"
+    t.bigint "supermarket_id", null: false
```

Comment:
Your customer table actually does NOT need a customer ID. It will have no foreign keys, but will be associated to items just through the joins table.

Reply:
Eventually, you can add the additional association between supermarkets and customers (and use the `has_many :customers, through: :items` that I mentioned above) but it's not necessary for these user stories.